src/dart_workspace_symbol_provider.ts

```diff
@@ -15,11 +15,11 @@ export class DartWorkspaceSymbolProvider implements WorkspaceSymbolProvider {
 		let chars = Array.from(query);
 		// Filter out regex special chars.
 		chars = chars.filter((c) => {
-				return '[]()\\-'.indexOf(c) == -1;
-    });
+			return '[]()\\-'.indexOf(c) == -1;
+		});
 		chars = chars.map((c: string) => {
-				if (c.toUpperCase() == c.toLowerCase()) return c;
-				return `[${c.toUpperCase()}${c.toLowerCase()}]`;
+			if (c.toUpperCase() == c.toLowerCase()) return c;
+			return `[${c.toUpperCase()}${c.toLowerCase()}]`;
 		});
```

Comment:
Whoops, I totally misread this - I blame the lack of side-by-side diffs (despite never using them in even in Kiln ;))